react/Search.tsx

```diff
@@ -24,45 +27,66 @@ const Search: FC<Props> = ({ query }) => {
     variables: { searchQuery: queryString },
   })
 
+  const { isMobile } = useDevice()
+
   const results: [SearchResult] = prop('searchEngine', data) || []
 
   const isNotLastResult = (results: [SearchResult], index: number) =>
     index !== results.length - 1
 
-  if (loading) return null
+  const isFirstResult = (index: number) => index === 0
+
+  if (results.length < 1 && isMobile) return <SearchBar />
 
   return (
     <Fragment>
-      <h1 className="t-heading-1 center">
-        {results.length > 0
-          ? `Results for "${queryString}"`
-          : 'No Results Found'}
-      </h1>
-      <ul className="list pl0 center mt5 mw8">
-        {results
-          ? results.map((result: SearchResult, index: number) => (
-              <li
-                key={index}
-                className={`mh8 mv8 searchResult ${
-                  isNotLastResult(results, index) ? 'searchResultBorder' : ''
-                }`}>
-                <h2 className="t-heading-2 searchTitle">{result.title}</h2>
-                <p className="t-body searchSnippet">{result.snippet}</p>
-                <a
-                  className={`link c-emphasis no-underline t-body ml-auto
+      {isMobile ? <SearchBar /> : ''}
```

Comment:
This could just be `isMobile && <SearchBar />`

Reply:
Done